src/ILLink.CodeFix/RequiresUnreferencedCodeCodeFixProvider.cs

```diff
@@ -19,7 +20,7 @@ namespace ILLink.CodeFix
 	public class RequiresUnreferencedCodeCodeFixProvider : BaseAttributeCodeFixProvider
 	{
 		public sealed override ImmutableArray<string> FixableDiagnosticIds
-			=> ImmutableArray.Create (RequiresUnreferencedCodeAnalyzer.IL2026);
+			=> ImmutableArray.Create (DiagnosticId.RequiresUnreferencedCode.AsString ());
```

Comment:
Consider storing the `SupportedDiagnostics` of the `RequiresUnreferencedCodeAnalyzer` as a static property, then calling that property to get the full list of diagnostic IDs for this.

Same idea for `RequiresAssemblyFilesCodeFixProvider`.